packages/dataviews/src/components/dataform-context/index.tsx

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Added a DataForm context to store the field definitions. This way I can more easily manage the layout on a field level without having to make sure I drill down the field definitions.